seed/challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -660,13 +660,13 @@
         "<code>    }</code>",
         "<code>}</code>",
         "We can remove the function keyword and colon (:) altogether - and get this:",
-        " const Container extends Component {",
-        "     render() {",
-        "       return {",
-        "         `<div>Container</div>`",
-        "       }",
-        "     }",
-        " }",
+        "<code> const Container extends Component { </code>",
```

Comment:
This multi-line example in the description of the challenge should be in `<blockquote>` tags. [Here's](https://github.com/freeCodeCamp/freeCodeCamp/blob/15474e8065d633657842a3208b1e6ca34814abf6/seed/challenges/02-javascript-algorithms-and-data-structures/object-oriented-programming.json#L45) an example of it being used. You can use `<br>` for line breaks. Make sure those spaces are there to retain the formatting.